File: TrabajoAcademicoNet/SoftProgDBManager/Db2DBManagerFactory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PUCP.Inf30.SoftProg.DB
{
    public class Db2DBManagerFactory : DBManagerFactory
    {
        public override DBManager CrearDBManager(String host, int puerto, String esquema,
            String usuario, String password)
        {
            return Db2DBManager.getInstance(host, puerto, esquema, usuario, password);
        }
    }
}

File: Java/TrabajoAcademico/SoftProgDBManager/src/main/java/pe/edu/pucp/inf30/softprog/dbmanager/Db2DBManagerFactory.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package pe.edu.pucp.inf30.softprog.dbmanager;

/**
 *
 * @author Cristhian Horacio
 */
public class Db2DBManagerFactory extends DBManagerFactory {

    @Override
    public DBManager crearDBManager(String host, int puerto, String esquema,
            String usuario, String password) {
        return Db2DBManager.getInstance(host, puerto, esquema, usuario, password);
    }
    
}